### PR TITLE
(packaging) Ship azure_inventory plugin and update bundled modules

### PR DESCRIPTION
**Pull request by donoghuc**
This commit pulls in the azure_inventory module. In addition other bundled modules are updated to the latest version.

### CODE REVIEW

File: Puppetfile [hunk 1]
```diff
@@ -23,11 +23,14 @@ mod 'puppetlabs-zone_core', '1.0.2'
...
 # Useful additional modules
 mod 'puppetlabs-package', '0.6.0'
-mod 'puppetlabs-puppet_conf', '0.3.1'
+mod 'puppetlabs-puppet_conf', '0.4.0'
 mod 'puppetlabs-python_task_helper', '0.3.0'
 mod 'puppetlabs-reboot', '2.2.0'
 mod 'puppetlabs-ruby_task_helper', '0.4.0'
 
+# Plugin modules
+#mod 'puppetlabs-azure_inventory', '0.2.0'
```
Comment (donoghuc) [inline Puppetfile]: @nicklewis We need to figure out why the push to forge job is failing. https://jenkins-master-prod-1.delivery.puppetlabs.net/job/forge-module_puppetlabs-azure_inventory_init-tag_push-forge/3/console

**Comment by donoghuc** [comment]
Another packaging PR https://github.com/puppetlabs/puppet-runtime/pull/229

File: Puppetfile (hunk 1 above)
Comment (donoghuc) [reply Puppetfile]: Alex pointed out that it is not that important to publish this module to the forge. Pulling from git tag for now.

File: lib/bolt/cli.rb [hunk 2]
```diff
@@ -544,7 +544,7 @@ def bundled_content
       # We only need to enumerate bundled content when running a task or plan
       content = { 'Plan' => [],
                   'Task' => [],
-                  'Plugin' => %w[puppetdb pkcs7 prompt terraform task] }
+                  'Plugin' => Bolt::Plugin::BUILTIN_PLUGINS }
```
Comment (donoghuc) [inline lib/bolt/cli.rb]: Do we also want to append the 'azure_inventory` module here?

Comment (donoghuc) [reply lib/bolt/cli.rb]: Yes, eventually we will want to move most of these to modules (notably *not* pdb or prompt).

File: Puppetfile [hunk 3]
```diff
@@ -23,11 +23,16 @@ mod 'puppetlabs-zone_core', '1.0.2'
...
 # Useful additional modules
 mod 'puppetlabs-package', '0.6.0'
-mod 'puppetlabs-puppet_conf', '0.3.1'
+mod 'puppetlabs-puppet_conf', '0.4.0'
 mod 'puppetlabs-python_task_helper', '0.3.0'
 mod 'puppetlabs-reboot', '2.2.0'
 mod 'puppetlabs-ruby_task_helper', '0.4.0'
 
+# Plugin modules
+mod 'puppetlabs-azure_inventory',
```
Comment (nicklewis) [inline Puppetfile]: I pushed 0.2.0 up manually, so we should be able to pull it from the forge now.

Comment (donoghuc) [reply Puppetfile]: nice, Updated.

**Pull request merged**